verilog/formatting/formatter_test.cc

```diff
@@ -878,8 +878,8 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "output reg yyy //c2\n"
      " ) ;endmodule:foo\n",
      "module foo (\n"
-     "    input  wire x,  //c1\n"   // aligned
-     "    output reg  yyy  //c2\n"  // trailing comments not aligned
+     "    input  wire x,   //c1\n"  // aligned
+     "    output reg  yyy  //c2\n"  // aligned
```

Comment:
Will it also work with `/* these type of comments */`?
Please add regular tests with such comments at the end of line and maybe one test when a multi line comment is actually occupying multiple lines to make sure the formatter doesn't crash.

Reply:
Added handling of comment blocks `/* */ ` and tests for them.
Aligning multiline comments doesn't work however it doesn't cause crash.